### PR TITLE
Adding swedish language

### PR DESCRIPTION
**Pull request by eragnms**
I have added the Swedish language and tests for it.

/M

### CODE REVIEW

File: tests/test_languages.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 
 from tests import BaseTestCase
 
-
```
Comment (waqasshabbir) [inline tests/test_languages.py]: double blank lines are recommended between imports and class definition. Please revert this change.

File: dateparser/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.6.0'
+__version__ = '0.7.0.dev1'
```
Comment (eragnms) [inline dateparser/__init__.py]: Not sure about the procedures for pull requests, but I believe I have reverted as you proposed.
Thanks for the comment!

File: tests/test_languages.py (hunk 1 above)
Comment (eragnms) [inline tests/test_languages.py]: Updated...
Thanks
